dlk/hw/make_os/build/make_rootfs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+	echo "Error: Number of argument should be 1"
+	exit 0
+fi
+
+BIT_32_OR_64=$1
```

Comment:
Same as `make command`, arg is better to be an architecture, `armhf`, `arm64`.
`$ make_rootfs.sh armhf` looks good for me better than `$ make_rootfs.sh 32`

Reply:
I'll take your idea and use same expression.